cocos/core/animation/animation-state.ts

```diff
@@ -382,6 +385,7 @@ export class AnimationState extends Playable {
             min: 0,
             max: this._clip.duration,
         };
+        this._playbackDuration = clip.duration;
```

Comment:
I think should initialize invDuration too

Reply:
This may be a BUG. I didn't do it here on purpose since `duration` was also not initialized here(`duration` was initialized in `initialze()`). I'm intend to keep them sync.